pkg/controller/servicemesh/memberroll/controller.go

```diff
@@ -556,6 +556,10 @@ func (r *namespaceReconciler) initializeNetworkingStrategy() error {
 	clusterNetwork.SetKind("ClusterNetwork")
 	err := r.client.Get(context.TODO(), client.ObjectKey{Name: "default"}, clusterNetwork)
 	if err != nil {
+		if errors.IsNotFound(err) {
```

Comment:
Is this Kind supposed to exist in 3.11? If not, we should be checking for `meta.IsNoMatchError(err)` as well?

Reply:
Yes.  This is available in 3.11 and 4.x.  I don't know what conditions are required for the resource not to exist.